core/site.go

```diff
@@ -514,8 +514,16 @@ func (site *Site) collectMeters(key string, meters []config.Device[api.Meter]) [
 		}
 
 		props := deviceProperties(dev)
+		title := props.Title
+		if title == "" {
+			if t, ok := dev.Config().Other["template"].(string); ok {
```

Comment:
Aua. Können wir das in eine Funktion der Config kapseln? Wenn wir das machen sollten wir die auch im Logging konsequent statt name verwenden, siehe auch https://github.com/evcc-io/evcc/issues/20336.

Reply:
Kommen wir hier auch an 'Name'? Das wäre natürlich noch besser als Templatename und oder Typ.
Könnten Name auch als dediziertes Feld published und die fallbacklogik rein im UI halten

Reply:
Nein, name gibts zumindest in der yaml nur vorher. Bei DB kennt die Config das.

Reply:
Schade, das wäre noch schöner.